src/sql/workbench/api/common/extHostDataProtocol.ts

```diff
@@ -572,6 +572,10 @@ export class ExtHostDataProtocol extends ExtHostDataProtocolShape {
 		return this._resolveProvider<azdata.BackupProvider>(handle).getBackupConfigInfo(connectionUri);
 	}
 
+	public override $createSas(handle: number, ownerUri: string, blobContainerUri: string, blobContainerKey: string, storageAccountName: string, expirationDate: string): Thenable<azdata.CreateSasResponse> {
```

Comment:
This doesn't belong here - Data Protocol is only for the common provider functionality (what is defined in sqlops-dataprotocolclient)

You already have the extHostAzureBlob which will act as the proxy for this. Look at mainThreadAzureAccount for how that's hooked up (it injects itself directly into the service)